packages/cwd-pre-propose-base/src/error.rs

```diff
@@ -22,6 +22,9 @@ pub enum PreProposeError {
     #[error("Message sender is not dao")]
     NotDao {},
 
+    #[error("Message is not a pause or remove schedule message ")]
+    NotAPauseOrRemoveScheduleMsg {},
```

Comment:
Let's make it an "invalid proposal" or "malformed proposal" mb? Especially since we add it to the "base" package. It looks very specific for a single contract here in this form